### PR TITLE
Feat/faq suggestions

### PR DESCRIPTION
**Pull request by SavvyShah**
[Notion card](https://www.notion.so/reactivesearch/FAQs-SearchBox-React-Vue-15b41f2c96da4bb1b640687e648969f8)

Adds FAQ suggestions which can be used to provide instant answers to frequent user queries instead of waiting for network requests. Loom preview attached in notion comments.

- [x] [ReactiveCore PR](https://github.com/appbaseio/reactivecore/pull/162)
- [x] [Vue Playground](https://github.com/appbaseio/vue-playground/pull/68)
- [x] [React Playground](https://github.com/appbaseio/playground/pull/122)

**Learn more about contributing:** [Contributing guides](https://github.com/appbaseio/reactivesearch/blob/next/.github/CONTRIBUTING.md)

### CODE REVIEW

File: packages/vue/src/components/search/SearchBox.jsx [hunk 1]
```diff
@@ -478,11 +487,7 @@ const SearchBox = defineComponent({
 						this.AIResponse
 						&& this.AIResponse.response
 						&& this.AIResponse.response.question,
-					answer:
-						this.AIResponse
-						&& this.AIResponse.response
-						&& this.AIResponse.response.answer
-						&& this.AIResponse.response.answer.text,
+					answer: this.mergedAIAnswerc,
```
Comment (SavvyShah) [inline packages/vue/src/components/search/SearchBox.jsx]: Typo here

File: packages/vue/src/components/search/SearchBox.jsx [hunk 2]
```diff
@@ -478,11 +487,7 @@ const SearchBox = defineComponent({
 						this.AIResponse
 						&& this.AIResponse.response
 						&& this.AIResponse.response.question,
```
Comment (SavvyShah) [inline packages/vue/src/components/search/SearchBox.jsx]: Pass suggestion as question here.

File: packages/vue/src/components/search/SearchBox.jsx [hunk 3]
```diff
@@ -1790,6 +1809,11 @@ const SearchBox = defineComponent({
 													)}
 													placeholder={this.$props.placeholder}
 													autoFocus={this.$props.autoFocus}
+													showFocusShortcutsIcon={
```
Comment (SavvyShah) [inline packages/vue/src/components/search/SearchBox.jsx]: showFocusShortcutsIcon, showVoiceSearch shouldn't be added.

File: packages/web/src/components/search/SearchBox.js [hunk 4]
```diff
@@ -481,6 +486,24 @@ const SearchBox = (props) => {
...
+			setValue(
+				suggestion.value,
+				true,
+				props,
+				causes.SUGGESTION_SELECT,
+				true,
+				false,
+				suggestion._category,
+				false,
+				false,
```
Comment (SavvyShah) [inline packages/web/src/components/search/SearchBox.js:502]: Make changes same as vue.

Comment (SavvyShah) [inline packages/web/src/components/search/SearchBox.js:502]: Make changes same as vue.

**Review by mohdashraf010897 — APPROVED** [review]

**Pull request merged**